distributed-job-manager/src/main/java/ru/fix/distributed/job/manager/Worker.java

```diff
@@ -573,6 +582,7 @@ public void close() throws Exception {
 
         workShareLockService.close();
 
+        detachProfilerIndicators();
```

Comment:
In main profiler implementation - AggregatingProfiler - it just removes indicators from ConcurrentHashMap. But I suppose it's better to detach indicators according to profiler documentation:
```
//During service destruction
profiler.dettachIndicator("buffer.size");
```